src/GitTools.IssueTrackers/FodyWeavers.xml

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Weavers>
+    <Costura>
+        <IncludeAssemblies>
+            Jira.SDK
+        </IncludeAssemblies>
+    </Costura>
+  <JetBrainsAnnotations />
```

Comment:
We should add the annotations file as a source file (with internal accessibility). That way any annotations we use can be leveraged by consumers of this library